### ISSUE TITLE
Help Wanted: Create A Delete Profile Button

### ISSUE THREAD

**Issue by justynth** [justynth/Friend_Builder#129]
With Mr. Zuckerberg recently coming under fire for putting people's personal information at risk, we have decided that our project could use a delete profile button on the edit profile screen so that users can delete their information from this app whenever they choose.
Tasks:
- Create a button on the edit profile screen that will allow a user to delete his profile

**Comment by pavelekpl** [comment]
Hi, is this project still alive? I'd like to contribute to it, thanks.

**Comment by justynth** [comment]
I don't believe that anyone is currently doing much to maintain this project, you're more than welcome to contribute to the code though.
